source/common/grpc/async_client_impl.h

```diff
@@ -25,13 +26,14 @@ class AsyncClientImpl final : public AsyncClient {
   AsyncStream* start(const Protobuf::MethodDescriptor& service_method,
                      AsyncStreamCallbacks& callbacks) override;
 
-  TimeSource& timeSource() { return cm_.timeSource(); }
+  TimeSource& timeSource() { return time_source_; }
```

Comment:
Still a bit puzzled why we need to have a getter for `TimeSource` here..

Reply:
source/common/grpc/async_client_impl.cc:211 needs this, but I see that class is already a friend, and in this code we already have the pattern of friends digging into private member vars without going thru a private method. So I can remove it.